<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enviar E-mail Produtividade</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Produtividade</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Anexar a planilha:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chamados Atendidos:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="input_chamados_atendidos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>WO0000000 INSTALAÇÃO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chamados Não Atendidos:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_nao">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Não</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="input_chamados_nao_atendidos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>WO0000000 INSTALAÇÃO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_sim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sim</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEnviarEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar Email</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>input_chamados_atendidos</tabstop>
  <tabstop>radio_sim</tabstop>
  <tabstop>radio_nao</tabstop>
  <tabstop>input_chamados_nao_atendidos</tabstop>
  <tabstop>buttonEnviarEmail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
